### PR TITLE
Add RDoc about scope option on distance_of_time_in_words

### PR DESCRIPTION
**Pull request by maurogeorge**

### CODE REVIEW

**Comment by zzak** [comment]
IMO this seems like a bit verbose for the API docs.. I wonder if this would fit better in the i18n guide?

**Comment by maurogeorge** [comment]
Hi @zzak not sure if verbose is a problem. If we take a look at the [migration RDoc](http://api.rubyonrails.org/classes/ActiveRecord/Migration.html) for example we have lot of content too.

IMO search something on the RDoc, I expect to see the full documentation, options etc including options that maybe not be on the guides.

What you think about rewrote changing the yml to something like:

```
      #   datetime:
      #     distance_in_words:
      #       short:
      #         about_x_hours:
      #           one: 1 hr
      #           other: '%{count} hr'
```

and add a link to the [i18n file](https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/en.yml) when we have all the keys.

**Comment by zzak** [comment]
Yes. Let's reduce the duplication in the `yml` for this document, link to the full file, and also limit the number of examples used.

**Comment by maurogeorge** [comment]
@zzak updated could you please review? Thanks :smile:

File: actionview/lib/action_view/helpers/date_helper.rb [hunk 1]
```diff
@@ -68,6 +68,26 @@ module DateHelper
       #   distance_of_time_in_words(from_time, to_time, include_seconds: true)                        # => about 6 years
       #   distance_of_time_in_words(to_time, from_time, include_seconds: true)                        # => about 6 years
       #   distance_of_time_in_words(Time.now, Time.now)                                               # => less than a minute
+      #
+      # With the <tt>scope</tt> you can define a custom scope to rails lookup
```
Comment (zzak) [inline actionview/lib/action_view/helpers/date_helper.rb]: "..scope for Rails to lookup the translation."

File: actionview/lib/action_view/helpers/date_helper.rb [hunk 2]
```diff
@@ -68,6 +68,26 @@ module DateHelper
...
+      # for the translation.
+      # For example you can define the following in your locale (e.g. en.yml)
```
Comment (zzak) [inline actionview/lib/action_view/helpers/date_helper.rb]: nitpick: "... (e.g. `en.yml`)."

File: actionview/lib/action_view/helpers/date_helper.rb [hunk 3]
```diff
@@ -68,6 +68,26 @@ module DateHelper
...
+      #
+      #   datetime:
+      #     distance_in_words:
+      #       short:
+      #         about_x_hours:
+      #           one: 1 hr
+      #           other: '%{count} hr'
+      #
+      # See https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/en.yml
+      # for the complete I18n file.
```
Comment (zzak) [inline actionview/lib/action_view/helpers/date_helper.rb]: "...for more examples."

File: actionview/lib/action_view/helpers/date_helper.rb [hunk 4]
```diff
@@ -68,6 +68,26 @@ module DateHelper
...
+      #
+      # Which then will result in
```
Comment (zzak) [inline actionview/lib/action_view/helpers/date_helper.rb]: "..in the following:"

**Comment by maurogeorge** [comment]
@zzak updated with your changes, could you please review? Thanks

**Comment by zzak** [comment]
@maurogeorge Looks good, thanks for adjusting! <3

**Comment by maurogeorge** [comment]
@zzak thanks for the merge :green_heart:

**Pull request merged**